pulp_docker/app/serializers.py

```diff
@@ -54,3 +60,125 @@ class Meta:
     class Meta:
         fields = platform.PublisherSerializer.Meta.fields
         model = models.DockerPublisher
+
+
+class DockerDistributionSerializer(platform.ModelSerializer):
```

Comment:
Seems like we could save some code by inhering from the platform.DistributionSerializer. If there is a problem with that, maybe a comment?

Reply:
The the base_url is on a Distribution and registry_path is on a DockerDistribution ... this makes the serializers different enough that subclassing from Distribution would require removing the base_url field.